customersupport/templates/template.html

```diff
@@ -41,9 +41,9 @@
         </div>
         <div id="navbar" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
-                <li class="active"><a href="{{ url_for("new_call") }}">New Call</a></li>
+                <li><a href="{{ url_for("new_call") }}">New Call</a></li>
```

Comment:
Yea, I wasn't sure if the time would be well spent figuring out how to do this for a slight usability increase.